### PR TITLE
pettier now uses prettier to format lib

### PR DESCRIPTION
**Pull request by shermango**
iono

### CODE REVIEW

**Comment by coleturner** [comment]
LOL I love it

File: package.json [hunk 1]
```diff
@@ -5,5 +5,8 @@
   "main": "index.js",
   "repository": "https://github.com/coleturner/pettier.git",
   "author": "Cole Turner <turner.cole@gmail.com>",
-  "license": "MIT"
+  "license": "MIT",
+  "devDependencies": {
+    "prettier": "^1.13.5"
+  }
 }
```
Comment (coleturner) [inline package.json:12]: Would you mind adding a precommit hook for this?

**Review by coleturner — CHANGES_REQUESTED** [review]

**Comment by coleturner** [comment]
Also feel free to add yourself as a contributor / create a contributors section

File: package.json (hunk 1 above)
Comment (shermango) [reply package.json:12]: yea sure

**Review by coleturner — APPROVED** [review]

**Pull request merged**